README.md

```diff
@@ -60,9 +60,9 @@ If you are using a `self-hosted` Windows runner, `GNU tar` and `zstd` are requir
 
 ### Outputs
 
-* `cache-hit` - A boolean value to indicate an exact match was found for the key.
+* `cache-hit` - A string `'true'` or `'false'` indicating whether an exact match was found for the key.
 
-    > **Note** `cache-hit` will only be set to `true` when a cache hit occurs for the exact `key` match. For a partial key match via `restore-keys` or a cache miss, it will be set to `false`.
+    > **Note** `cache-hit` will only be set to `'true'` when a cache hit occurs for the exact `key` match. For a partial key match via `restore-keys` or a cache miss, it will be set to `'false'`.
```

Comment:
This is still incorrect. On a cache miss, the output is not set, i.e. empty string.

Reply:
@itchyny  So you are saying it is three-valued, `'true'`, `'false'` or ''?

Reply:
@andreasabel Exactly. A bit surprising behavior though. See https://github.com/itchyny/actions-cache-issues-1263/actions/runs/8385466326.

Reply:
Thanks for the thorough investigation @itchyny !  I updated the text accordingly.